requirements.txt

```diff
@@ -1 +1,3 @@
-flower
\ No newline at end of file
+# sudo pip install -r requirements.txt
+flower==0.7.3
```

Comment:
I thought they were at 0.9 or something?

Reply:
@johnclarkemills The version I tested with initially (0.9.2) hits this error `https://github.com/mher/flower/issues/343` which the author refuses to fix, unless you use oauth.

Reply:
0.7.3 is the latest version that does not suffer from that issue.